lib/utilities/monitoringHandler.js

```diff
@@ -64,6 +64,36 @@ const httpResponseSizeBytes = new client.Summary({
     help: 'Cloudserver HTTP response sizes in bytes',
 });
 
+const quotaEvaluationDuration = new client.Histogram({
+    name: 's3_cloudserver_quota_evaluation_duration_seconds',
+    help: 'Duration of the quota evaluation operation',
+    labelNames: ['action', 'code', 'type'],
```

Comment:
do we need to filter by `code` or `action` ? i got mixed feeling here:
* we don't really care about the action with regard to the _duration_, since all these actions should be affected the same way?
* the code (= result of the evaluation) is definitely not relevant for the _duration_
* however, it could indeed be interesting to have this labels to allow checking which requests cause the most quota checks or the quota check success/rate
* but at the same time, we can probably get this information by just looking at the request count (which has action & code).... the only thing missing from the request count may be the `type` of quota check (i.e. none, bucket, account, bucket+account)

Reply:
Depending on the API (write vs delete) and the response (quota exceeded vs passing), both action and code are relevant to discriminate between the different flows of the quota evaluation logic
For example quota exceeded will have more impact as we need to clean the inflights in case of quota exceeded